filebeat/input/filestream/identifier_test_non_windows.go

```diff
@@ -0,0 +1,96 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+// +build !windows
+
+package filestream
+
+import (
+	"io/ioutil"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	loginp "github.com/elastic/beats/v7/filebeat/input/filestream/internal/input-logfile"
+	"github.com/elastic/beats/v7/libbeat/common"
+	"github.com/elastic/beats/v7/libbeat/common/file"
+)
+
+func TestFileIdentifierInodeMarker(t *testing.T) {
+	t.Run("inode_marker file identifier", func(t *testing.T) {
+		const markerContents = "unique_marker"
+		markerFile, err := ioutil.TempFile("", "test_file_identifier_inode_marker_identifier")
+		if err != nil {
+			t.Fatalf("cannot create marker file for test: %v", err)
+		}
+		defer os.Remove(markerFile.Name())
+
+		_, err = markerFile.Write([]byte(markerContents))
+		if err != nil {
+			t.Fatalf("cannot write marker contents to file: %v", err)
+		}
+		markerFile.Sync()
+
+		c := common.MustNewConfigFrom(map[string]interface{}{
+			"identifier": map[string]interface{}{
+				"inode_marker": map[string]interface{}{
+					"path": markerFile.Name(),
```

Comment:
btw. you can save some lines here by writing `"identifier.inode_marker.path": markerFile.Name()`